### ISSUE TITLE
Gracefully handle webhook errors

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1575]
For example, when a webhook.site URL is deleted. It responds with a 404. This actually crashes the process when `eventMode: true` (it is `true` by default now).

Eventually, the process will need to retry failed requests via config options - e.g `retryOnWebhookFailure`

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.10.14](https://github.com/open-wa/wa-automate-nodejs/compare/3.10.13...3.10.14) (2021-04-13)

- build(deps): bump optional-require from 1.0.2 to 1.0.3 [`#1576`](https://github.com/open-wa/wa-automate-nodejs/pull/1576)
- build(deps-dev): bump marked from 2.0.1 to 2.0.3 [`#1577`](https://github.com/open-wa/wa-automate-nodejs/pull/1577)
- build(deps-dev): bump release-it from 14.5.1 to 14.6.1 [`#1578`](https://github.com/open-wa/wa-automate-nodejs/pull/1578)
- build(deps-dev): bump eslint from 7.23.0 to 7.24.0 [`#1579`](https://github.com/open-wa/wa-automate-nodejs/pull/1579)
- build(deps): bump ts-json-schema-generator from 0.89.2 to 0.89.3 [`#1580`](https://github.com/open-wa/wa-automate-nodejs/pull/1580)
- Release 3.10.14 [`84b7674`](https://github.com/open-wa/wa-automate-nodejs/commit/84b76748932978eb8ac2221bea030732b9a33e73)
- 🐛  fix ws reverse proxy detection on popup [`00ed003`](https://github.com/open-wa/wa-automate-nodejs/commit/00ed00376bf6677169777c3c5ca3a4061d72dd3c)
- ✨ feat: added `batteryLow` to `client.healthCheck` [`966bbfc`](https://github.com/open-wa/wa-automate-nodejs/commit/966bbfc793c55f9a8a30f3b85e4017527ce72df6)
- 🐛  ignore webhook errors for now #1575 [`d211eb5`](https://github.com/open-wa/wa-automate-nodejs/commit/d211eb55e572c82a2c8a9baa2abb6ece1f7bd2f7)

**Comment by smashah** [comment]
complete retries and backoff later #659
